public/index.html

```diff
@@ -49,10 +49,14 @@
         <a href="https://www.terrestris.de/en/impressum/">
           Imprint
         </a>
-        –
+        —
         <a href="https://www.terrestris.de/en/datenschutzerklaerung">
           Privacy Policy
         </a>
+        —
+        <a href="https://github.com/geostyler/geostyler/releases/tag/v<%= htmlWebpackPlugin.options.geostylerVersion %>">
+          GeoStyler <%= htmlWebpackPlugin.options.geostylerVersion %>
```

Comment:
```suggestion
          GeoStyler v<%= htmlWebpackPlugin.options.geostylerVersion %>
```

Maybe we also want to link to the release page?: https://github.com/geostyler/geostyler/releases/tag/v10.1.0